### PR TITLE
Make applab use es6 import/export syntax everywhere

### PR DESCRIPTION
**Pull request by pcardune**

### CODE REVIEW

**Comment by caleybrock** [comment]
I think this is awesome and saves me from doing it when I touch files. Couldn't find any errors from looking over the changed files, but would definitely wait for circle. LGTM

File: apps/src/applab/ChartApi.js [hunk 1]
```diff
@@ -4,9 +4,10 @@
  * For now, uses Google charts.
  * @see {GoogleChart}
  */
- /* global Promise */
-var utils = require('../utils');
-var GoogleChart = require('./GoogleChart');
+/* global Promise */
+import * as utils from '../utils';
+
+import GoogleChart from './GoogleChart';
```
Comment (islemaster) [inline apps/src/applab/ChartApi.js:10]: What's with the whitespace?

File: apps/src/applab/CompletionButton.jsx [hunk 2]
```diff
@@ -1,9 +1,8 @@
...
-
-var React = require('react');
-var connect = require('react-redux').connect;
-var ProtectedStatefulDiv = require('../templates/ProtectedStatefulDiv');
-var experiments = require('../experiments');
+import msg from '@cdo/locale';
+import React from 'react';
+import {connect} from 'react-redux';
+import ProtectedStatefulDiv from '../templates/ProtectedStatefulDiv';
+import experiments from '../experiments';
```
Comment (islemaster) [inline apps/src/applab/CompletionButton.jsx:5]: experiments.js isn't using ES6 export syntax yet. Any possible problems here?

File: apps/src/applab/Exporter.js [hunk 3]
```diff
@@ -1,15 +1,14 @@
...
-var download = require('../assetManagement/download');
-var elementLibrary = require('./designElements/library');
-var exportProjectEjs = require('../templates/exportProject.html.ejs');
-var exportProjectReadmeEjs = require('../templates/exportProjectReadme.md.ejs');
-var logToCloud = require('../logToCloud');
+import JSZip from 'jszip';
+import {saveAs} from 'filesaver.js';
+import assetPrefix from '../assetManagement/assetPrefix';
+import download from '../assetManagement/download';
+import elementLibrary from './designElements/library';
```
Comment (islemaster) [inline apps/src/applab/Exporter.js:8]: Likewise some of these.  assetPrefix.js also has multiple exports old-style.  I remember this causing some issues before we converted utils.js to export ES6-style.

**Comment by pcardune** [comment]
@islemaster We've been using es6 style imports for files with es5 style exports for a while now.

The main issue is that when we eventually upgrade those files to use es5 exports, we will have to go to all the files that import them and make sure they are using `import *`.

i.e.,

```
import assetPrefix from '../assetManagement/assetPrefix';
```

will have to turn into

```
import * as assetPrefix from '../assetManagement/assetPrefix';
```

onces `assetPrefix.js` gets converted to es6 imports.

As part of using `lebab`, you have to go through and fix all the places where you need to use `import *` as `lebab` unfortunately does not do that for you.

In any case, assuming all the tests pass, I think these changes are safe. And these changes aren't adding any extra work down the road that we don't already have to do anyway.

**Comment by islemaster** [comment]
Well, apps tests succeeded.  LGTM.

**Pull request merged**